src/main/java/com/community/gateway/jwt/config/UserDetailsServiceImpl.java

```diff
@@ -21,7 +21,7 @@
 	public UserDetails loadUserByUsername(String mobileNumber) throws UsernameNotFoundException {
 		OperatorDTO user = null;
 		try {
-			user = operatorLogical.findByMobile(Integer.parseInt(mobileNumber));
+			user = operatorLogical.findByMobile(mobileNumber);
```

Comment:
Mobile number should be long / String. For now I changed to String. Feel free to use the data type you want based on business use-case.